iam/api/QuickStartNewTest/QuickStartNewTest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using Xunit;
+
+public class QuickStartNewTest : IDisposable
+{
+    private readonly string _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+    private ServiceAccount serviceAccount;
+    private IamService iamService;
+
+    public QuickStartNewTest()
+    {
+        // Create service account for test
+        var credential = GoogleCredential.GetApplicationDefault()
+            .CreateScoped(IamService.Scope.CloudPlatform);
+        iamService = new IamService(
+            new IamService.Initializer
+            {
+                HttpClientInitializer = credential
+            });
+
+        var request = new CreateServiceAccountRequest
+        {
+            AccountId = "iam-test-account",

```

Comment:
Maybe add a random part to this name, something based on DateTime.UTCNow so that in case these tests are run more than once at the same time they don't fail.